### PR TITLE
Video listing Page UI only

### PR DESCRIPTION
**Pull request by Ankur9669**
Files to check:
1. VideoListing.js
2. fetch-products.js
3. Video.js
4. videolisting.css

image

### CODE REVIEW

File: FitVid/src/Pages/VideoListing/videolisting.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    cursor: pointer;
+    background-color: rgb(49, 47, 47);
+}
+.videolisting-video{
+    width: 100%;
+    height: 100%;
+}
+.thumbnail-text-container{
+    display: grid;
+    grid-template-columns: 50px auto;
```
Comment (Sahil25061999) [inline FitVid/src/Pages/VideoListing/videolisting.css]: use rem here.

**Review by Sahil25061999 — APPROVED** [review]
code looks good to me, proper conventions have been followed, requires some minor changes related to rem -px conversion in css files. Couldn't suggest much ui change as the preview link is not working, but from the screenshot uploaded i would suggest to use the complete the complete red border around the home link

File: FitVid/src/Pages/VideoListing/videolisting.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    margin: 1rem;
+}
+.videolisting-video-container{
+    width: 20rem;
+    margin: 0.5rem;
+    cursor: pointer;
+    background-color: rgb(49, 47, 47);
+}
```
Comment (swstica) [inline FitVid/src/Pages/VideoListing/videolisting.css]: use var instead of rgb.

File: FitVid/src/Pages/VideoListing/videolisting.css [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    position: absolute;
+    top: 0.3rem;
+    right: -20px;
+    font-size: 1.5rem;
+    cursor: pointer;
+}
+.video-modal{
+    position: absolute;
+    top: 1.9rem;
+    right: -20px;
```
Comment (swstica) [inline FitVid/src/Pages/VideoListing/videolisting.css:58]: You can use rem instead of -20px

**Review by swstica — APPROVED** [review]
The code seems fine to merge! Just small changes are suggested otherwise:
Has good Readability 
Proper conventions

The preview link is not working but looking at the picture, I'd suggest you add border on right side also.

File: FitVid/src/Pages/VideoListing/videolisting.css (hunk 1 above)
Comment (Ankur9669) [reply FitVid/src/Pages/VideoListing/videolisting.css]: Done

File: FitVid/src/Pages/VideoListing/videolisting.css (hunk 2 above)
Comment (Ankur9669) [reply FitVid/src/Pages/VideoListing/videolisting.css]: Done

**Pull request merged**